### ISSUE TITLE
VideoStream from CCTV

### ISSUE THREAD

**Issue by lankoestee** [holoviz/panel#6926]
When I use the `VideoStream` class to get the network camera information, it can only get the video stream information of the local camera. If I want to use a remote network camera, such as a video stream transmitted through the rtmp or rtsp protocol, how should I display it in the panel?

Using opencv can achieve the effect I want:

```python
import cv2
cap = cv2.VideoCapture("rtsp://cctv_url")
ret, frame = cap.read()
while ret:
    ret, frame = cap.read()
    cv2.imshow("frame",frame)
    if cv2.waitKey(1) & 0xFF == ord('q'):
        break
    
cv2.destroyAllWindows()
cap.release()

```

Is it possible to use Panel's `VideoStream` to achieve the above effect?

**Comment by MarcSkovMadsen** [comment]
Yes. I've tried something like that by streaming the frame as pngs to the PNG pane.

**Comment by lankoestee** [comment]
@MarcSkovMadsen Thanks! The [tutorial](https://github.com/holoviz/panel/blob/enhancement/intermediate-streaming-tutorial/doc/tutorials/intermediate/build_server_video_stream.md) you mentioned is very useful! However, the video display fps is very slow. When I use the `cv2.imshow` method, I can reach a display frame rate of 15fps, but the frame rate is significantly lower when using the [tutorial](https://github.com/holoviz/panel/blob/enhancement/intermediate-streaming-tutorial/doc/tutorials/intermediate/build_server_video_stream.md) method. Is there any better way to solve this problem?
